apps/glusterfs/app_test.go

```diff
@@ -75,12 +75,14 @@ func TestAppAdvsettings(t *testing.T) {
 	defer os.Remove(dbfile)
 	os.Setenv("HEKETI_EXECUTOR", "mock")
 	defer os.Unsetenv("HEKETI_EXECUTOR")
+	os.Setenv("HEKETI_DB_PATH", dbfile)
+	defer os.Unsetenv("HEKETI_DB_PATH")
 
 	data := []byte(`{
 		"glusterfs" : {
 			"executor" : "crazyexec",
 			"allocator" : "simple",
-			"db" : "` + dbfile + `",
+			"db" : "/path/to/nonexistent/heketi.db",
```

Comment:
This is potentially problematic: Despite its name, his path may exist by coincidence...

Also, since you don't seem to specifically test the db setting here, why change this at all?

Reply:
Test case has added, please see below.